MetaMask/metamask-extension
# Move HW wallet tracking in Swaps

## Description
Move HW wallet tracking outside of `sensitiveProperties` in Swaps.

## Manual testing steps
  - Open background.html and go to the Network tab
  - Filter it by the `batch` string
  - Check that we send these 2 params now inside the `properties` field
